modules/apps/powwow/powwow-portlet/docroot/meetings/css/main.css

```diff
@@ -1,8 +1,11 @@
 @import "compass";
 @import "mixins";
 
-.powwow-portlet {
+.lfr-search-container {
```

Comment:
Is there any reason or needs that led you to modify this CSS? As of CSS, it will effect in the whole page, that may be not our expectation. So, if need to, we can use another CSS selector, or wrap with the portlet identifier.